prow/kube/prowjob.go

```diff
@@ -79,6 +79,8 @@ type ProwJobStatus struct {
 	URL             string       `json:"url,omitempty"`
 	PodName         string       `json:"pod_name,omitempty"`
 	BuildID         string       `json:"build_id,omitempty"`
+	UUID            string       `json:"uuid,omitempty"`
```

Comment:
Not clear why would we want the new field since the prowjob Name is guaranteed to be unique. Also, I was referring to pj.Metadata.UID, which is set by the k8s api server during the creation of the ProwJob.